account_payment_order/models/account_payment_order.py

```diff
@@ -465,6 +465,10 @@ def _prepare_move_line_offsetting_account(
         else:
             account_id = self.journal_id.payment_debit_account_id.id
 
+        if self.payment_mode_id.offsetting_account == "bank_account":
```

Comment:
offsetting_account field is not available in https://github.com/ursais/bank-payment/blob/14.0-imp-account_payment_order/account_payment_order/models/account_payment_mode.py#L78, so we are getting error.  Please check and make changes

It was removed in this commit(https://github.com/ursais/bank-payment/commit/cbe1eed5ac2a7987a3f23db1d975b6e13abab32c#diff-bc1e10412f0a69eab3d4e969864e38c2051ae55ee7a4f6fd10d1463e8ae9e72dL78)

cc: @SodexisTeam